assignments/stretch-function-conversion.js

```diff
@@ -1,23 +1,29 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
 // let myFunction = function () {};
+let myFunction = () => {};
 
 // let anotherFunction = function (param) {
 //   return param;
 // };
+let anotherFunction = (param) => param
```

Comment:
You can drop the parentheses here since you only have one parameter and aren't doing anything fancy with it:
```javascript
 let anotherFunction = param => param ;
```